src/Microsoft.TestPlatform.ObjectModel/RunSettings/TestRunSettings.cs

```diff
@@ -46,6 +45,7 @@ protected TestRunSettings(string name)
     /// Converter the setting to be an XmlElement.
     /// </summary>
     /// <returns>The Xml element for the run settings provided.</returns>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1059:MembersShouldNotExposeCertainConcreteTypes", Justification = "XmlElement is required in the data collector.")]
```

Comment:
That's unrelated but I noticed this was removed during the reformatting and should not have been.